phnt/include/ntldr.h

```diff
@@ -1384,57 +1448,20 @@ LdrSetImplicitPathOptions(
     );
 #endif
 
-#if (PHNT_VERSION >= PHNT_WINDOWS_10)
-#ifdef PHNT_INLINE_TYPEDEFS
-/**
- * The LdrControlFlowGuardEnforced function checks if Control Flow Guard is enforced.
- *
- * @return BOOLEAN TRUE if Control Flow Guard is enforced, FALSE otherwise.
- */
-FORCEINLINE
-BOOLEAN
-NTAPI
-LdrControlFlowGuardEnforced(
-    VOID
-    )
-{
-    return LdrSystemDllInitBlock.CfgBitMap && (LdrSystemDllInitBlock.Flags & 1) == 0;
-}
-#else
-// rev
+#if (PHNT_VERSION >= PHNT_WINDOWS_10RS3)
+// private
 /**
  * The LdrControlFlowGuardEnforced function checks if Control Flow Guard is enforced.
  *
  * @return BOOLEAN TRUE if Control Flow Guard is enforced, FALSE otherwise.
  */
 NTSYSAPI
-BOOLEAN
+ULONG
 NTAPI
 LdrControlFlowGuardEnforced(
     VOID
     );
 #endif
-#endif
-
-#if (PHNT_VERSION >= PHNT_WINDOWS_10)
-/**
- * The LdrControlFlowGuardEnforcedWithExportSuppression function checks if Control Flow Guard is
- * enforced with export suppression.
- *
- * @return BOOLEAN TRUE if Control Flow Guard is enforced, FALSE otherwise.
- */
-FORCEINLINE
-BOOLEAN
-NTAPI
-LdrControlFlowGuardEnforcedWithExportSuppression(
-    VOID
-    )
-{
-    return LdrSystemDllInitBlock.CfgBitMap
-        && (LdrSystemDllInitBlock.Flags & 1) == 0
-        && (LdrSystemDllInitBlock.MitigationOptionsMap.Map[0] & 3) == 3; // PROCESS_CREATION_MITIGATION_POLICY_CONTROL_FLOW_GUARD_EXPORT_SUPPRESSION
```

Comment:
Yep, this should have been `BYTE5(LdrSystemDllInitBlock.MitigationOptionsMap.Map[0])`.
```
.text:0000000180021A44                 mov     rcx, cs:LdrSystemDllInitBlock.MitigationOptionsMap.Map
.text:0000000180021A4B                 shr     rcx, 28h
.text:0000000180021A4F                 and     cl, 3
.text:0000000180021A52                 cmp     cl, 3
```